scripts/build.sh

```diff
@@ -22,10 +22,14 @@ source scripts/shared.sh
 
 parse_std_args "$@"
 
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 137112412989.dkr.ecr.us-west-2.amazonaws.com
+
 echo "building image ${version} ... "
 docker build \
     -f ${build_context}/docker/Dockerfile.${processor} \
     -t ${repository}:${version} \
     --build-arg REGION=${REGION} \
     -t sagemaker-spark:latest \
     ${build_context}
+
+docker logout https://137112412989.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
I recommend putting this URI, which is the same as the Image URI, in a bash variable.